x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/effected_policy_select/effected_policy_select.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  EuiFormRow,
+  EuiSelectable,
+  EuiSelectableProps,
+  EuiSwitch,
+  EuiSwitchProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EuiSelectableOption } from '@elastic/eui/src/components/selectable/selectable_option';
+import { PolicyData } from '../../../../../../../common/endpoint/types';
+
+interface OptionPolicyData {
+  policy: PolicyData;
+}
+
+type EffectedPolicyOption = EuiSelectableOption<OptionPolicyData>;
+
+export interface EffectedPolicySelection {
+  isGlobal: boolean;
+  selected: PolicyData[];
+}
+
+export type EffectedPolicySelectProps = Omit<
+  EuiSelectableProps<OptionPolicyData>,
+  'onChange' | 'options' | 'children' | 'searchable'
+> & {
+  options: PolicyData[];
+  isGlobal: boolean;
+  onChange: (selection: EffectedPolicySelection) => void;
+  selected?: PolicyData[];
+};
+export const EffectedPolicySelect = memo<EffectedPolicySelectProps>(
+  ({
+    isGlobal,
+    onChange,
+    listProps,
+    options,
+    selected = [],
+    'data-test-subj': dataTestSubj,
+    ...otherSelectableProps
+  }) => {
+    const DEFAULT_LIST_PROPS = useMemo(() => ({ bordered: true }), []);
```

Comment:
Any reason to not make this a constant outside of component?

Reply:
Good point. I probably just forgot to move it up to be a readonly const. Will do that. Thanks